app/controllers/invitations_controller.rb

```diff
@@ -1,8 +1,8 @@
 class InvitationsController < ApplicationController
-  before_filter :authorize
 
   # xhr only, tested in a request spec
   def create
+    authorise :invitation, :create?
```

Comment:
Should this be spelled `authorize`, or should all the other ones be spelled `authorise`?

I'm not familiar with pundit. Why does this one need a `:create?` argument while the other ones seem to magically infer the action?